kibana_prometheus_exporter/kibana_collector.py

```diff
@@ -150,17 +150,30 @@ def _process(self) -> Iterator[Metric]:
 
 
 class KibanaCollector(object):
-    def __init__(self, host: str, path: str = "/api/status", kibana_login: str = None, kibana_password: str = None):
+    def __init__(
+        self,
+        host: str,
+        path: str = "/api/status",
+        kibana_login: str = None,
+        kibana_password: str = None,
+        ignore_ssl: str = None,
```

Comment:
This should be a bool, as `_check_ssl` returns a bool and it's used as such in `_fetch_stats`.
As the expected default is to validate certificates, the default value should be `False`.